database/rrdhost.c

```diff
@@ -271,11 +276,15 @@ RRDHOST *rrdhost_create(const char *hostname,
         health_alarm_log_open(host);
     }
 
-    if (host->rrd_memory_mode == RRD_MEMORY_MODE_DBENGINE) {
+    if (host && host->rrd_memory_mode == RRD_MEMORY_MODE_DBENGINE) {
```

Comment:
Why do we change this? Just above there is `    if(host->health_enabled) {` which would crash if `host` was NULL.

Reply:
No need I will remove it. The initialization moved up from a previous commit and it could have ended up being NULL